docs/CANDIDATE_TECHNOLOGIES.md

```diff
@@ -0,0 +1,159 @@
+
+So, the factors that should be considered for selecting the candidate technology are:
+
+1. Community's adaptability and familiarity with the technology.
+2. Fulfillment of the project's requirements:
+   - Accept and read inputs from the gamepad.
+   - Relay functional code/script into the browser.
+   - Compatible across multiple platforms and devices.
+3. Stability and long-term support for the technology.
+4. Assist in implementing navigation features.
+
+## Candidate Technologies
+
+Before searching for a technology, I framed multiple approaches using which project requirements can be fulfilled. With those approaches in mind, I searched and looked through various technologies. Below is the list of the shortlisted ones:
+
+1. Custom Browser Extension / Add-on
+   - [Chrome Extension](https://developer.chrome.com/extensions)
+   - [Firefox Add-on](https://extensionworkshop.com/)
+   - [Safari Extension](https://developer.apple.com/library/archive/documentation/Tools/Conceptual/SafariExtensionGuide/Introduction/Introduction.html)
+2. Electron.js Desktop App with the following Node.js packages:
+   - [Node Key Sender](https://www.npmjs.com/package/node-key-sender) or [Robot.js](https://www.npmjs.com/package/robotjs)
+   - [Gamepad](https://www.npmjs.com/package/gamepad) (Alternative to Gamepad API for Node.js)
+3. [Espruino Pico](https://www.espruino.com/USB)
+4. Secondary Extensions to inject Scripts
+   - [Tampermonkey](https://chrome.google.com/webstore/detail/tampermonkey/dhdgffkkebhmkfjojejmpbldmpobfkfo?hl=en) (For Safari and Chromium-based browsers)
+   - [Greasemonkey](https://addons.mozilla.org/en-US/firefox/addon/greasemonkey/) (for Firefox)
+5. [Bookmarklets](https://support.mozilla.org/en-US/kb/bookmarklets-perform-common-web-page-tasks)
+
+## Analysis
+
+### Custom Browser Extension / Add-on
+
+This option would involve creating our browser extension (or add-on) that will enable the users to navigate using the gamepad.
+
+#### Advantages
+
+- Stable and long-term support.
+- Based on JavaScript and can support Infusion.
+- Provides various Chrome APIs which would be useful in implementing certain navigation effects/features.
+- Lightweight and easy for users to set-up (and install).
+- Requires no separate execution of scripts by the user.
+- Can run on both Desktop and Mobile (supported for a few browsers only).
+
+#### Disadvantages
+
+- Cross-browser restrictions, i.e., a Chrome-based extension can't run on Firefox or vice-versa.
+
+Most of the browsers (Chrome, Microsoft Edge, Opera, etc.) are based on Chromium and can thus run Chrome Extension. The Chromium-based browsers for mobile such as [Kiwi](https://play.google.com/store/apps/details?id=com.kiwibrowser.browser&hl=en_IN) also support and allow users to install and run Chrome Extensions (unlike the Google Chrome for mobile).
```

Comment:
Just "Google Chrome", "the" isn't necessary.

Reply:
Similarly, "Most browsers" is just fine.

Reply:
I have reworded it in my previous commit [here](https://github.com/fluid-lab/gamepad-navigator/pull/4/files#diff-b53d05f39c28591f46c423b3aa9c65b5R70)